apps/ui/src/helpers/aliases.json

```
@@ -0,0 +1,81 @@
+{
+  "treasuredao.eth": "treasuregaming.eth",
+  "cryptopugz": "cryptopugz.eth",
+  "aave": "aave.eth",
+  "aavegotchi": "aavegotchi.eth",
+  "airswap": "vote.airswap.eth",
+  "dhedge": "gov.dhedge.eth",
+  "unfederalreserve": "unfederalreserve.eth",
+  "andrewladdusaw.eth": "everydaydao.eth",
+  "sushi": "sushigov.eth",
+  "sushipowah": "sushigov.eth",
+  "yam": "yam.eth",
+  "yamv2": "yam.eth",
+  "velotoken": "velotoken.eth",
+  "cream": "cream-finance.eth",
+  "bzx": "bzx.eth",
+  "metafactory": "metafactory.eth",
+  "stabilize": "stabilize-governance.eth",
+  "diadata": "diadao.eth",
+  "gnosis": "gnosis.eth",
+  "farm": "harvestfi.eth",
+  "tokenlon": "tokenlon.eth",
+  "synthetixcouncil": "spartancouncil.eth",
+  "synthetixproposal": "snxgov.eth",
+  "synthetixgrants": "snxgrants.eth",
+  "cover": "cover-protocol.eth",
+  "iwan": "iwan.eth",
+  "oceandao": "officialoceandao.eth",
+  "macaronswap": "macaronswap.eth",
+  "pepemon": "pepedontdump.eth",
+  "akropolis-delphi": "akropolis.eth",
+  "rally": "rallygov.eth",
+  "debaseonomics": "debaseonomics.eth",
+  "pickle": "pickle.eth",
+  "index": "index-coop.eth",
+  "streamr": "streamr.eth",
+  "rarible": "rarible.eth",
+  "curve": "curve.eth",
+  "sil": "sister-in-law.eth",
+  "digital-reserve-currency": "drctoken.eth",
+  "web3-api": "polywrap.eth",
+  "mstable": "mstablegovernance.eth",
+  "adex": "adex.eth",
+  "balancer": "balancer.eth",
+  "peripheralist.eth": "tiledao.eth",
+  "seen": "seenhaus.eth",
+  "kleros": "kleros.eth",
+  "piedao": "piedao.eth",
+  "dracula": "draculasucks.eth",
+  "pillar": "pillarwallet.eth",
+  "yaxis": "yaxis.eth",
+  "spaceswap": "spaceswap.eth",
+  "descience.eth": "people-dao.eth",
+  "dforce": "dforcenet.eth",
+  "rari": "fuse.eth",
+  "nft20.eth": "musedao.eth",
+  "tripscommunity": "tripscommunity.eth",
+  "deversifi.eth": "rhinofi.vote",
+  "cake.eth": "cakevote.eth",
+  "project-galaxy.eth": "gal.eth",
+  "epns.eth": "pushdao.eth",
+  "tptdao.eth": "itokenpocket.eth",
+  "tallyhodao.eth": "tahodao.eth",
+  "goodghosting.eth": "halofi.eth",
+  "arbitrum-odyssey.eth": "arbitrumfoundation.eth",
+  "pokt-network": "poktdao.eth",
+  "lensterapp.eth": "lenster.xyz",
+  "nexondao.eth": "eralend.eth",
+  "pancake": "cakevote.eth",
+  "gov.radicle.eth": "gov.radworks.eth",
+  "daoge.eth": "ownthedoge.eth",
+  "ownthedaoge.eth": "ownthedoge.eth",
+  "uniswap": "uniswapgovernance.eth",
+  "popcorn-snapshot.eth": "vaultcraft-snapshot.eth",
+  "kuma-inu.eth": "kumatokens.eth",
+  "x7community.eth": "x7finance.eth",
+  "redactedcartel.eth": "dinero.xyz",
+  "gal.eth": "g-dao.eth",
+  "bzr.eth": "rwg.eth",
+  "superraredao.eth": "raredao.eth"
```

We should remove the redirection for SuperRare for now (they requested this)

```suggestion
  "bzr.eth": "rwg.eth"
```